README.md

```diff
@@ -78,6 +78,41 @@ export default function App() {
 }
 ```
 
+### Configuring the widget
+
+- `uiMessageWebviewUrlScheme`: Used in postMessages and OAuth redirects.
```

Comment:
I took this content from https://docs.mx.com/api#connect_request_a_url. I thought about just linking to that page but I wasn't sure how to explain that our api docs use underscore notation but this SDK uses camel case props, so I opted to include it here. Not sure how I feel about this.

Reply:
@inphomercial @sambev what do you guys think?

Reply:
I think it's cool to not link that page, since things are different enough that it probably isn't the most helpful.